README.md

```diff
@@ -46,6 +48,8 @@ bundle exec middleman server
 
 See your website on `http://localhost:4567` in your browser.
 
+If you experience [this issue with ffi](https://github.com/alphagov/tech-docs-gem/issues/254), possible fixes can be found [here](#issue-with-ffi-on-osx-mohave).
```

Comment:
If you experience the [FFI gem issue for Mojave users](https://github.com/alphagov/tech-docs-gem/issues/254), you should refer to this [list of possible fixes](#issue-with-ffi-on-osx-mohave).